golos.publication/objects.hpp

```diff
@@ -110,6 +110,14 @@ struct rewardrules {
     //uint64_t cashout_time; //TODO:
 };
 
+struct forumprops_record {
+    forumprops_record() = default;
+
+    bool socialon = true;
```

Comment:
Here, should be the name of the social-contract.
We shouldn't direct link them together.

Reply:
@afalaleev Why do we need name of contract here?
And if name - why only 1 contract?
Let's remember golosd config.ini: "plugin = plugin1 plugin2 plugin2"
It should be a vector I think.

Reply:
Because it is flexible.